app/download/views.py

```diff
@@ -114,6 +114,17 @@ def get_document_metadata(service_id, document_id):
 @download_blueprint.route("/services/<uuid:service_id>/documents/<uuid:document_id>/authenticate", methods=["POST"])
 def authenticate_access_to_document(service_id, document_id):
     key = request.json.get("key")
+    if redis_store.exceeded_rate_limit(
```

Comment:
Might be worth us putting in place a similar if statement that enables us to run doc download api locally without redis enabled like we do in the API (mostly for consistency, maybe it will be easier when we potentially have everything spun up with a docker compose to start redis alongside it).

https://github.com/alphagov/notifications-api/blob/a81f34f8bd6a3fa44d358563464ae545e53ce758/app/notifications/validators.py#L53

Reply:
If we switch to having REDIS_ENABLED=False for test and dev environments, then we don't need anything extra here - `redis_client.exceeded_rate_limit` automatically short circuits if REDIS_ENABLED is false.